Makefile.toml

```diff
@@ -0,0 +1,107 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = "true"
+CARGO_MAKE_WORKSPACE_EMULATION = true
+CARGO_MAKE_CRATE_WORKSPACE_MEMBERS = [
+	"crt0stack",
+	"enarx-keep-sgx",
+	"enarx-keep-sgx-shim",
+	"enumerate",
+	"intel-types",
+	"iocuddle",
+	"iocuddle-sgx",
+	"memory",
+	"nolibc",
+	"payload",
+	"sev",
+	"sev-show",
+	"sev-types",
+	"sgx-crypto",
+	"sgx-show",
+	"sgx-types",
+	"span",
+	"testing",
+	"units",
+	"vdso",
+	"vmsyscall"
+]
+
+[tasks.audit]
+install_crate = "cargo-audit"
+command = "cargo"
+args = ["audit"]
+
+[tasks.deny]
+install_crate = "cargo-deny"
+command = "cargo"
+args = ["deny", "check", "licenses"]
+
+[tasks.test]
+command = "cargo"
+args = ["test"]
+
+[tasks.check-format]
+command = "cargo"
+args = ["fmt", "--", "--check"]
+
+[tasks.misc-lints-missing-docs]
+workspace = false
+command = ".tests/misc-lints-missing-docs"
+
+[tasks.cargo-toml-package-edition]
+workspace = false
+# It would appear the 'install_crate' feature only works
+# if it is a cargo plugin... so we'll work around that.
+install_script = [
+'''
+which toml || cargo install toml-cli
+'''
+]
+command = ".tests/cargo-toml-package-edition"
+
+[tasks.misc-lints-clippy-all]
+workspace = false
+command = ".tests/misc-lints-clippy-all"
+
+[tasks.misc-licenses-rs-spdx]
+workspace = false
+command = ".tests/misc-licenses-rs-spdx"
+
+[tasks.misc-licenses-asm-spdx]
+workspace = false
+command = ".tests/misc-licenses-asm-spdx"
+
+[tasks.misc-diagrams]
+workspace = false
+command = ".tests/misc-diagrams"
+
+[tasks.cargo-toml-package-license]
+workspace = false
+command = ".tests/cargo-toml-package-license"
+
+[tasks.misc-licenses-crate]
+workspace = false
+command = ".tests/misc-licenses-crate"
+
+# A convenient target for running the entire continuous integration
+# suite locally
+[tasks.ci]
+# This is a top-level task, the other targets will implicitly
+# "opt-in" to being a workspace-level target meaning that cargo
+# make will descend into member crates automatically.
+workspace = false
+run_task = [{ name = [
+		"misc-lints-missing-docs",
+		"cargo-toml-package-edition",
+		"misc-lints-clippy-all",
+		"misc-licenses-rs-spdx",
+		"misc-licenses-asm-spdx",
+		"misc-diagrams",
+		"cargo-toml-package-license",
+		"misc-licenses-crate",
+		"check-format",
+		"deny",
+		"audit",
+		"clippy",
+		"build",
+		"test"
+] }]
```

Comment:
This is really handy. :+1:

Reply:
This section seems to me to be patching `tasks.ci` just in order to add additional custom tests. But I think [there is already a hook for that](https://github.com/sagiegurari/cargo-make/blob/master/src/lib/descriptor/makefiles/rust.toml#L394). If so, using the hook would be better because we can then pick up upstream changes to `tasks.ci` automatically.